packages/ui/src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { ChangeEvent, ReactText } from 'react';
+import { Checkbox as ChakraCheckbox, forwardRef, HTMLChakraProps, UseCheckboxProps } from '@chakra-ui/react';
+
+// We need to explicitly overwrite aria-invalid as forwardRef tries to put there values that Chakra doesn't support
+type Overwrites = Pick<UseCheckboxProps, 'aria-invalid'>;
+
+export interface CheckboxProps extends Omit<HTMLChakraProps<'div'>, keyof UseCheckboxProps>, Overwrites {
+  children?: ReactText | ReactText[];
+  value?: string | number;
+  onChange: (value: ChangeEvent<HTMLInputElement>) => void;
+  checked: boolean;
+  name?: string;
+  disabled?: boolean;
+  size?: 'md' | 'lg';
+}
+
+export const Checkbox = forwardRef<CheckboxProps, 'input'>(
+  ({ checked, disabled = false, size = 'md', ...props }, ref) => {
+    return (
+      <ChakraCheckbox
+        ref={ref}
+        borderRadius={2}
```

Comment:
You can specify `borderRadius` and the default `colorScheme` on a theme level. This way we can have single source of truth when it comes to component styles